src/cowboy_swagger_redirect_handler.erl

```diff
@@ -8,7 +8,7 @@
 %% Handlers
 -export([resource_exists/2, previously_existed/2, moved_permanently/2]).
 
--type state() :: #{}.
+-type state() :: {file, string()}.
```

Comment:
Why not `#{file := string()}`?

Reply:
Just use what was already passed. So this is a fix for type spec.  Dialyzer is happy now.